### PR TITLE
BACKLOG-13954, BACKLOG-13980: implement remove of automatic list orde…

### PR DESCRIPTION
**Pull request by jkevan**

### CODE REVIEW

File: src/javascript/EditPanel/EditPanelContent/FormBuilder/ChildrenSection/AutomaticOrdering/AutomaticOrdering.jsx [hunk 1]
```diff
@@ -8,11 +8,29 @@ import {Button} from '@jahia/moonstone';
 import {useTranslation} from 'react-i18next';
 import {adaptSectionToDisplayableRows} from './AutomaticOrdering.utils';
 import {getDisplayedRows} from './AutomaticOrdering.utils';
+import {withStyles} from '@material-ui/core';
+import {Clear} from '@material-ui/icons';
```
Comment (Taybou) [inline src/javascript/EditPanel/EditPanelContent/FormBuilder/ChildrenSection/AutomaticOrdering/AutomaticOrdering.jsx:12]: moonstone doesn't provide this icon?

File: src/javascript/EditPanel/EditPanelContent/FormBuilder/ChildrenSection/AutomaticOrdering/AutomaticOrdering.jsx [hunk 2]
```diff
@@ -8,11 +8,29 @@ import {Button} from '@jahia/moonstone';
...
+import {IconButton} from '@jahia/design-system-kit';
 
-export const AutomaticOrderingCmp = ({formik: {values}}) => {
+const styles = theme => ({
```
Comment (Taybou) [inline src/javascript/EditPanel/EditPanelContent/FormBuilder/ChildrenSection/AutomaticOrdering/AutomaticOrdering.jsx:15]: it's great if you can move the style part to scss file, it's more easy to maintain

File: src/javascript/EditPanel/EditPanelContent/FormBuilder/ChildrenSection/ChildrenSection.jsx [hunk 3]
```diff
@@ -1,14 +1,15 @@
...
-import {Typography, Toggle} from '@jahia/design-system-kit';
+import {Typography, Toggle, Badge} from '@jahia/design-system-kit';
 import {ChildrenSectionPropTypes} from '~/FormDefinitions/FormData.proptypes';
 import {ManualOrdering} from './ManualOrdering';
 import {useTranslation} from 'react-i18next';
 import {AutomaticOrdering} from './AutomaticOrdering';
 import {Constants} from '~/ContentEditor.constants';
 import {compose} from '~/utils';
 import {connect} from 'formik';
+import {Public} from '@material-ui/icons';
```
Comment (Taybou) [inline src/javascript/EditPanel/EditPanelContent/FormBuilder/ChildrenSection/ChildrenSection.jsx:12]: same for this icon

File: src/javascript/EditPanel/EditPanelContent/FormBuilder/ChildrenSection/AutomaticOrdering/AutomaticOrdering.jsx [hunk 4]
```diff
@@ -24,16 +42,61 @@ export const AutomaticOrderingCmp = ({formik: {values}}) => {
...
+        setFieldTouched(displayedRowToRemove.propField.name, true);
+        setFieldValue(displayedRowToRemove.directionField.name, undefined, true);
+        setFieldTouched(displayedRowToRemove.directionField.name, true);
+    };
+
+    const getInputContext = (index, isDirection) => {
+        const inputContext = {
+            displayBadges: false,
+            displayLabels: index === 0,
+            setActionContext: () => {}
```
Comment (Taybou) [inline src/javascript/EditPanel/EditPanelContent/FormBuilder/ChildrenSection/AutomaticOrdering/AutomaticOrdering.jsx:63]: I guess it's not used this function

Comment (jkevan) [reply src/javascript/EditPanel/EditPanelContent/FormBuilder/ChildrenSection/AutomaticOrdering/AutomaticOrdering.jsx:63]: it is

File: src/javascript/EditPanel/EditPanelContent/FormBuilder/ChildrenSection/ChildrenSection.jsx (hunk 3 above)
Comment (jkevan) [reply src/javascript/EditPanel/EditPanelContent/FormBuilder/ChildrenSection/ChildrenSection.jsx:12]: dont know I reuse the same as the one used for the same badge in Field.jsx

File: src/javascript/EditPanel/EditPanelContent/FormBuilder/ChildrenSection/AutomaticOrdering/AutomaticOrdering.jsx (hunk 2 above)
Comment (jkevan) [reply src/javascript/EditPanel/EditPanelContent/FormBuilder/ChildrenSection/AutomaticOrdering/AutomaticOrdering.jsx:15]: will do in next PR

File: src/javascript/EditPanel/EditPanelContent/FormBuilder/ChildrenSection/AutomaticOrdering/AutomaticOrdering.jsx (hunk 1 above)
Comment (jkevan) [reply src/javascript/EditPanel/EditPanelContent/FormBuilder/ChildrenSection/AutomaticOrdering/AutomaticOrdering.jsx:12]: dont know, will check in next PR

**Pull request merged**